libcircle.spec

```diff
@@ -0,0 +1,191 @@
+Name:    libcircle
+%global maj_ver 0.3
+Version: %{maj_ver}.0
+Release: 1%{?dist}
+
+Source: https://github.com/hpc/%{name}/releases/download/v%{maj_ver}/%{name}-%{version}.tar.gz
+URL: http://hpc.github.io/libcircle/
+Summary: A library used to distribute workloads
+License: BSD
+
+%global openmpi_ver openmpi3
+
+%if 0%{?fedora}
+%global openmpi_ver openmpi
+%endif
+
+BuildRequires:  check-devel
+BuildRequires:  doxygen
+BuildRequires:  graphviz
+
+%description
+A simple interface for processing workloads using an automatically
+distributed global queue.
+
+%package %{openmpi_ver}
+Summary:        Libcircle Open MPI libraries
+BuildRequires:  %{openmpi_ver}-devel
+
+%description %{openmpi_ver}
+A simple interface for processing workloads using an automatically
+distributed global queue.
+
+libcircle compiled with Open MPI
+
+%package mpich
+Summary:        Libcircle MPICH libraries
+BuildRequires:  mpich-devel environment-modules
```

Comment:
We are standardising on `lmod` (and `lua-lmod` for Leap) rather than `environment-modules`.  See the hdf5-vol-daos packaging for details.